contracts/sol6/wrappers/KyberRateHelper.sol

```diff
@@ -172,11 +163,11 @@ contract KyberRateHelper is IKyberRateHelper, WithdrawableNoModifiers, Utils5 {
         uint256 optionalSellAmount,
         uint256 networkFeeBps
     ) internal view returns (bytes32[] memory sellReserves, uint256[] memory sellRates) {
-        Amounts memory A;
+        Amounts memory amts;
```

Comment:
might as well call it amounts
amts is non conventional.
as bad as A

Reply:
Fixed